website/docs/d/vm.html.markdown

```diff
@@ -0,0 +1,74 @@
+---
+layout: "vcd"
+page_title: "VMware Cloud Director: vcd_vm"
+sidebar_current: "docs-vcd-datasource-vm"
+description: |-
+  Provides a VMware Cloud Director VM data source. This can be used to access standalone VMs.
+---
+
+# vcd\_vm
+
+Provides a VMware Cloud Director standalone VM data source. This can be used to access standalone VMs.
+
+Supported in provider *v3.2+*
+
+## Example Usage
+
+```hcl
+data "vcd_vm" "web1" {
+  name = "web1"
+}
+```
+
+## Arguments and attributes reference
+
+This data source provides all arguments and attributes available for `vcd_vapp_vm`, with the only difference that the
+`vapp_name` is not required.
+
```

Comment:
Please add the usual bullet point on the argument, which would quickly state that name is either name or ID. Something like:

`name - (Required) A name or ID for the standalone VM in VDC`

Reply:
added